tests/results/TextPositionTest.py

```diff
@@ -36,23 +36,48 @@ def test_comparison(self):
         uut2 = TextPosition(None, None)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 > uut2)
+        self.assertFalse(uut1 < uut2)
 
         uut1 = TextPosition(1, 2)
         uut2 = TextPosition(1, 3)
         self.assertFalse(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertTrue(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
 
         uut1 = TextPosition(2, None)
         uut2 = TextPosition(3, None)
         self.assertFalse(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 > uut2)
+        self.assertTrue(uut1 < uut2)
 
         uut1 = TextPosition(None, None)
         uut2 = TextPosition(4, 5)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
 
         uut1 = TextPosition(4, 8)
         uut2 = TextPosition(4, 8)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut2 <= uut1)
+        self.assertFalse(uut1 > uut2)
+        self.assertFalse(uut2 < uut1)
```

Comment:
@aptrishu @Makman2 And don't forget `!=`. It's always good to have this test too...